pytorch_to_returnn/torch/cuda.py

```diff
@@ -0,0 +1,8 @@
+def is_available():
+  """
+  Always returns False for now
+
+  :return: False
+  :rtupe: bool
+  """

```

Comment:
Bad documentation.
Typo.
You are redundant. Use `:return:` to describe what you return, but then don't write it again twice as the description.

But anyway, this is wrong. We probably will change this. You should not document as if that is how it should be. If you really would want a documentation, write sth like "Dummy/stub. Not implemented yet...". Or just leave it away.

```suggestion
```